src/NServiceBus.AcceptanceTesting/Support/EndpointRunner.cs

```diff
@@ -173,13 +173,13 @@ void RegisterInheritanceHierarchyOfContextInSettings(ScenarioContext context)
 
         async Task Cleanup()
         {
-            dynamic transportCleaner;
+            IConfigureTestExecution transportCleaner;
             if (busConfiguration.GetSettings().TryGet("CleanupTransport", out transportCleaner))
             {
                 await transportCleaner.Cleanup();
             }
 
-            dynamic persistenceCleaner;
+            IConfigureTestExecution persistenceCleaner;
             if (busConfiguration.GetSettings().TryGet("CleanupPersistence", out persistenceCleaner))
```

Comment:
This could potentially expand to builders/serializers too. Don't think execution order matters. I'll make the change.